packages/eslint-plugin-slow-imports/lib/rules/no-top-level-external-import.js

```diff
@@ -80,9 +82,20 @@ module.exports = {
       }
 
       if (!isExternalModule(modulePath)) {
-        const absoluteModulePath = resolve(modulePath, context);
+        const absoluteModulePath = relative(
+          modulePath,
+          filename,
+          context.settings
+        );
+
+        if (!absoluteModulePath) {
+          context.report({
+            node: originalNode,
+            messageId: "CANNOT_RESOLVE_MODULE",
+          });
```

Comment:
Shouldn't the function return after this?

Reply:
I think so. Good catch.